manifests/t/TeamUTY/UndertaleYellow/1.1.0/TeamUTY.UndertaleYellow.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TeamUTY.UndertaleYellow
+PackageVersion: 1.1.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Undertale Yellow v1_1\Undertale Yellow.exe
+Installers:
+- InstallerUrl: https://download.gamejolt.net/01564098c7597e3525a229ffe576b2ada2768ed91040c7489cf1fbf56568bb91,1703892925,7/data/games/2/175/136925/files/657b82d7d3859/undertale-yellow-v1.1.zip

```

Comment:
Hi @Exorcism0666,

I got a slightly different path: https://download.gamejolt.net/3defa062b0613b393767b440dcab79e0dbe7602701cad073a2538924628dd66d,1703955918,7/data/games/2/175/136925/files/657b82d7d3859/undertale-yellow-v1.1.zip

It's the same pattern, which implies these might be time-based URLs with the time component hashed. What do you think?

Reply:
I think so! but I don't know how to go about putting the link there :c